src/cargo/util/toml/mod.rs

```diff
@@ -178,25 +178,26 @@ impl<'de> de::Deserialize<'de> for TomlDependency {
 }
 
 #[derive(Deserialize, Serialize, Clone, Debug, Default)]
+#[serde(rename_all = "kebab-case")]
```

Comment:
Ah unfortuantely I don't think we can do this as it breaks manifests that use `default_features = false`, for example.

When we first moved to serde we ran into that (as the old rustc-serialize toml-rs accepted both) and that's why there's two entries below :(

Reply:
The `#[serde(rename = "default_features")]` down on default_features2 will handle that: https://play.rust-lang.org/?gist=db44b7c463a8b4ac43c0b67e2b36e5d0&version=stable

Reply:
Oh! I missed that, nice!